coupon-service/src/main/resources/application-batch-dev.yml

```
@@ -20,15 +23,18 @@ spring:
-  port: -1
-
-management:
-  tracing:
-    enabled: false
+logging:
+  level:
+    root: info
+    shop.genieus: debug
+    org.springframework.cloud.gateway: debug
+    org.springframework.web.reactive: info
+    reactor.netty: warn
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)